app/src/main/java/com/onboarding/calculator/MainActivity.java

```diff
@@ -1,9 +1,9 @@
 package com.onboarding.calculator;
 
-import androidx.appcompat.app.AppCompatActivity;
-
 import android.os.Bundle;
 
+import androidx.appcompat.app.AppCompatActivity;
+
 import com.onboarding.calculator.databinding.ActivityMainBinding;
 import com.onboarding.calculator.mvp.contract.CalculatorContract;
 import com.onboarding.calculator.mvp.model.CalculatorModel;
```

Comment:
you can use Ctrl + Alt + O command to optimize the imports

Reply:
Solved!